src/main/scala/chisel3/simulator/package.scala

```diff
@@ -156,7 +156,8 @@ package object simulator {
               }
           }
         }
-        DataMirror.modulePorts(dut).flatMap {
+        // Chisel ports can be Data or Property, but there is no ABI for Property ports, so we only return Data.
+        DataMirror.modulePorts(dut).collect { case (name, port: Data) => (name, port) }.flatMap {
```

Comment:
I'm not sure what this is doing, but it seems like we can ignore properties here.

Reply:
Could change the flatMap rather than adding a collect to make this slightly more performant.